mk/compile.mk

```diff
@@ -17,7 +17,7 @@ objs		:=
 # Disable all builtin rules
 .SUFFIXES:
 
-comp-cflags$(sm) = -std=gnu99
+comp-cflags$(sm) = $(cflags_compiler_$(COMPILER_$(sm))) -std=gnu99
```

Comment:
I think this should rather go with the other warns flags below.

Reply:
Right, and I will rename the variable to show it is warning flags and for consistency: `comp-cflags-warns-clang`.